File: 백준/Bronze/3052. 나머지/나머지.c
#include <stdio.h>

int main()
{
    int n, arr[10], answer = 0;
    
    for (int i = 0; i < 10; i++) {
        scanf("%d", &n);
        
        arr[i] = n % 42;
    }
    
    for (int i = 0; i < 10; i++) {
        int cnt = 0;
        
        for (int j = i + 1; j < 10; j++) {
            if (arr[i] == arr[j]) {
                cnt++;
                break;
            }
        }
        
        if (cnt == 0) answer++;
    }
    
    printf("%d", answer);
}

File: 백준/Bronze/3052. 나머지/나머지.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n, answer = 0;
		int[] arr = new int[10];
		
		for (int i = 0; i < 10; i++) arr[i] = sc.nextInt() % 42;
		
		for (int i = 0; i < 10; i++) {
		    int cnt = 0;
		    
		    for (int j = i + 1; j < 10; j++) {
		        if (arr[i] == arr[j]) {
		            cnt++;
		            break;
		        }
		    }
		    
		    if (cnt == 0) answer++;
		}
		
		System.out.println(answer);
	}
}